### PR TITLE
Fixing docstrings for cvt

### PR DESCRIPTION
**Pull request by AnugunjNaman**
# What does this PR do?

This PR does the following:
1. Remove the error in `README.md` where `CvT` description was copy of `CTRL`
2. Fix `size` of image for `feature extractor` which was set to `224`.
3. fix the input docstring for forward classes of `CvtModel` and `CvtForImageClassification` (head mask etc not needed).
4. Adding the largest `CvT` model. 

TODO: **Transfer the largest sized (1GB) model to Microsoft from anugunj/cvt-w24-384-22k**

@NielsRogge @LysandreJik

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -249,7 +249,7 @@ Current number of checkpoints: ![](https://img.shields.io/endpoint?url=https://h
...
 1. **[CTRL](https://huggingface.co/docs/transformers/model_doc/ctrl)** (from Salesforce) released with the paper [CTRL: A Conditional Transformer Language Model for Controllable Generation](https://arxiv.org/abs/1909.05858) by Nitish Shirish Keskar*, Bryan McCann*, Lav R. Varshney, Caiming Xiong and Richard Socher.
-1. **[CvT](https://huggingface.co/docs/transformers/main/model_doc/cvt)** (from Microsoft) released with the paper [CvT: Introducing Convolutions to Vision Transformers](https://arxiv.org/abs/1909.05858) by Nitish Shirish Keskar*, Bryan McCann*, Lav R. Varshney, Caiming Xiong and Richard Socher.
+1. **[CvT](https://huggingface.co/docs/transformers/main/model_doc/cvt)** (from Microsoft) released with the paper [CvT: Introducing Convolutions to Vision Transformers](https://arxiv.org/abs/2103.15808) by Haiping Wu, Bin Xiao, Noel Codella, Mengchen Liu, Xiyang Dai, Lu Yuan, Lei Zhang.
```
Comment (NielsRogge) [inline README.md:252]: You'll have to run `make fix-copies` again (and `make fixup`) after this.

File: src/transformers/generation_utils.py [hunk 2]
```diff
@@ -942,8 +942,8 @@ def generate(
                 If set to float < 1, only the most probable tokens with probabilities that add up to `top_p` or higher
                 are kept for generation.
             typical_p (`float`, *optional*, defaults to 1.0):
-                The amount of probability mass from the original distribution to be considered in typical decoding.
-                If set to 1.0 it takes no effect. See [this paper](https://arxiv.org/pdf/2202.00666.pdf) for more details.
+                The amount of probability mass from the original distribution to be considered in typical decoding. If
+                set to 1.0 it takes no effect. See [this paper](https://arxiv.org/pdf/2202.00666.pdf) for more details.
```
Comment (NielsRogge) [inline src/transformers/generation_utils.py:946]: Normally this PR shouldn't change this file.

**Review by NielsRogge — APPROVED** [review]
LGTM!

**Review by LysandreJik — APPROVED** [review]
Nice!

**Pull request closed without merging**